packages/inter-protocol/test/auction/README-bidding-cli.md

```diff
@@ -109,8 +112,15 @@ Using the `test` keyring is lower friction, but feel free to use the os keyring:
 agd keys add test-acct
 ```
 
+## Provisioning your account for use with Zoe smart contracts
+
+Mainnet
```

Comment:
This is a testing tool. Use with mainnet is outside the scope.

Please coordinate before pushing to this branch.

I made https://github.com/Agoric/agoric-sdk/tree/bot-bid-cli-doc and put your work there.